game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
I simply forgot we have a NetTable with current scores and the limit.
NetTables are better because if someone reconnects the NetTable repopulates. Via events the information is lost.